### PR TITLE
Add website docs for blueprint, catalog and deployment constructs

### PR DESCRIPTION
**Pull request by dmettem**
Add website docs for:
data 'vra_blueprint'
data 'vra_blueprint_version'
data 'vra_catalog_item'
data 'vra_catalog_source_blueprint'
data 'vra_catalog_source_entitlement'
data 'vra_deployment'

resource 'vra_blueprint'
resource 'vra_blueprint_version'
resource 'vra_catalog_source_blueprint'
resource 'vra_catalog_source_entitlement'
resource 'vra_content_source'
resource 'vra_deployment'

Signed-off-by: Deepak Mettem <dmettem@vmware.com>

### CODE REVIEW

File: website/docs/d/vra_blueprint.html.markdown [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+* `org_id` - The id of the organization this entity belongs to.
+
+* `project_name` - The name of the project the entity belongs to.
+
+* `self_link` - HATEOAS of the entity.
+
+* `status` - Status of the cloud template. Supported values: `DRAFT`, `VERSIONED`, `RELEASED`.
+
+* `request_scope_org` - Flag to indicate whether this blueprint can be requested from any project in the organization this entity belongs to.
```
Comment (wilsonandvmware) [inline website/docs/d/vra_blueprint.html.markdown:68]: r should be before s in order

Comment (dmettem) [reply website/docs/d/vra_blueprint.html.markdown:68]: Done.

File: website/docs/r/vra_content_source.html.markdown [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  }
+}
+```
+
+
+## Argument Reference
+
+* `name` - (Required) A human-friendly name for content source used as an identifier in APIs that support this option.
+
+* `type_id` - (Required) Content Source type. Supported values are `com.gitlab`, `com.github`, `com.vmware.marketplace`, `org.bitbucket`.
```
Comment (wilsonandvmware) [inline website/docs/r/vra_content_source.html.markdown:41]: t should be after p in order

**Review by wilsonandvmware — COMMENTED** [review]
two minor comments

File: website/docs/r/vra_content_source.html.markdown (hunk 2 above)
Comment (dmettem) [reply website/docs/r/vra_content_source.html.markdown]: Done.

**Comment by wilsonandvmware** [comment]
LGTM

**Review by wilsonandvmware — APPROVED** [review]
LGTM

**Review by zhizhu7 — APPROVED** [review]

**Pull request merged**